features_and_tools/restful_api.rst

```diff
@@ -41,6 +45,32 @@ If you allow for multiple deletes, you have to specify a maximum number of resou
         }
     }
 
+Update with no changes requires `UpdateNoOp` plugin to be enabled. 
+The plugin can be configured to ignore additional meta elements in the resource. 
+By default, when plugin is enabled the following meta elements are ignored during resource comparison: ``versionId``, ``lastUpdated`` and ``source`` You can add additional meta elements to be ignored
```

Comment:
Maybe good to explain why these meta-elements should be ignored

Reply:
doen